certify-mockdp-identity.properties

```diff
@@ -102,6 +103,7 @@ mosip.certify.key-values={\
                                   'locale': 'en', \
                                   'logo': {'url': 'https://${mosip.api.public.host}/inji/veridonia-logo.png','alt_text': 'a square logo of a MOSIP'},\
                                   'background_color': '#f8efea',\
+                                  'background_image': { 'uri': 'https://vishwa-vyom.github.io/DID/white-background.png' }, \
```

Comment:
Since the background image is not there, let's not add

Reply:
In the above also

Reply:
Removed the bg img from places where it was not previously there